scripts/project.py

```diff
@@ -0,0 +1,54 @@
+"""Module for deploying project contract."""
+from brownie import FELToken, ProjectContract, ProjectManager, accounts, config
+
+from scripts.crypto import decrypt_nacl, encrypt_nacl, export_public_key, get_current_secret
+
+
+def deploy_project(owner):
+    """Deploy project contract."""
+    token = FELToken[-1]
```

Comment:
if you want to use latest contract shouldn't it be `FELToken[0]` ?

Reply:
Maybe it is irrelevant, but in `map.json` https://eth-brownie.readthedocs.io/en/stable/deploy.html#the-deployment-map 
`The list for each contract is sorted by the block number of the deployment with the most recent deployment first.`

Reply:
But here (https://github.com/eth-brownie/brownie/issues/1003#issuecomment-802880435) they write that the contract object `FELToken` is list where `-1` index is the latest deployment. Here they also write that deployments are appended to the list:
https://eth-brownie.readthedocs.io/en/stable/core-contracts.html#deploying-contracts
There is no explicit word about correct indexing in documentation, so I am not sure.